### ISSUE TITLE
Add sensuctl subcommand for managing time windows of filters

### ISSUE THREAD

**Issue by palourde** [sensu/sensu-go#1121]
The `EventFilter` struct [supports time windows](https://github.com/sensu/sensu-go/blob/26da78fcd4c586228ee2b3276c77f2320024bdda/types/filter.pb.go#L32) but sensuctl lacks the ability to create or manage these time windows:

```
$ sensuctl filter -h
[...]
Commands:
  create      create new filters
  delete      delete filter given name
  info        show detailed filter information
  list        list filters
  update      update filters

$ sensuctl filter create -h
[...]
Options:
  -a, --action string         specifies whether events are passed through the filter or blocked by the filter. Allowed values: allow, deny
      --api-url string        host URL of Sensu installation
      --cache-dir string      path to directory containing cache & temporary files (default "/Users/splourde/Library/Caches/sensu/sensuctl")
      --config-dir string     path to directory containing configuration files (default "/Users/splourde/.config/sensu/sensuctl")
      --environment string    environment in which we perform actions (default "default")
  -h, --help                  help for create
      --interactive           Determines if CLI is in interactive mode
      --organization string   organization in which we perform actions (default "default")
  -s, --statements string     comma separated list of boolean expressions that are evaluated to determine if the event matches this filter
```

We need to add a subcommand to sensuctl so time windows can be supported. The implementation would probably be really similar to the check subdue subcommand: https://github.com/sensu/sensu-go/blob/master/cli/commands/check/subcommands/set_subdue.go.

Once implemented, we also need to update the 2.x filters documentation to reflect that change.

**Comment by grepory** [comment]
Please file a separate issue for sensu-docs to update the documentation.
